geoapps/octree_creation/application.py

```diff
@@ -27,7 +28,6 @@ class OctreeMesh(ObjectDataSelection):
     Widget used for the creation of an octree mesh
     """
 
-    defaults = {}
```

Comment:
Is this the line that the issue is supposed to address?  That would make sense to me if defaults were imported above and the {} wiped them out, but that doesn't seem to be the case

Reply:
Importing OctreeMesh would overwrite the base class defaults.